Packs/FeedAzure/Integrations/FeedAzure/README.md

```diff
@@ -0,0 +1,60 @@
+Azure.CloudIPs Feed Integration.
+This integration was integrated and tested with version xx of AzureFeed
```

Comment:
@ShahafBenYakir -  you need to add a version for Azure here

Reply:
Removing this line as there is no API for this feed (we only download a file from the [MS azure page](https://www.microsoft.com/en-us/download/confirmation.aspx?id=56519))